valhalla/midgard/util.h

```diff
@@ -21,6 +23,9 @@
 
 #define UNUSED(x) (void)(x)
 
+#define VALHALLA_UNLIKELY(x) BOOST_UNLIKELY(x)
```

Comment:
We can implement it on our own, but I decide that since we depend on boost anyway it is better to simply use it as alias for now...

Reply:
We could also put it in `src/config.h`? That's quite bit lighter and this could be used anywhere.